packages/coinstac-ui/app/main/index.js

```diff
@@ -124,6 +136,65 @@ loadConfig()
       mainWindow.webContents.send(BAD_TOKEN);
     });
 
+    /**
+     * IPC Listener to prepare consortia files
+     */
+    ipcMain.on('prepare-consortia-files', async (event, { userId, fileTree, appDirectory }) => {
+      const userRunDirectory = path.join(appDirectory, 'runs', userId);
+
+      if (!fs.existsSync(userRunDirectory)) {
+        fs.mkdirSync(userRunDirectory, { recursive: true });
+      }
+
+      const res = [];
+
+      fileTree.forEach((consortium) => {
+        const consortiumDirectory = path.join(userRunDirectory, consortium.name);
+
+        if (!fs.existsSync(consortiumDirectory)) {
+          fs.mkdirSync(consortiumDirectory);
+        }
+
+        consortium.runs.forEach((run) => {
+          const runDirectory = path.join(consortiumDirectory, `${run.pipelineName} - ${run.id}`);
+
+          if (!fs.existsSync(runDirectory)) {
+            fs.mkdirSync(runDirectory, { recursive: true });
+          }
+
+          const outputDirectory = path.join(appDirectory, 'output', userId, run.id);
+
+          if (fs.existsSync(outputDirectory)) {
+            const allFiles = getAllFilesInDirectory(outputDirectory);
+
+            allFiles.forEach((file) => {
+              const relativePath = file.replace(outputDirectory, '');
```

Comment:
`path.basename()`

Reply:
actually, you want this path relative to the `runId` dir so you would use `path.relative(from, to)`